benchmarks/timeseries/serializable_ringbuffer.py

```diff
@@ -0,0 +1,82 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Benchmarks the `SerializableRingbuffer` class."""
+
+from __future__ import annotations
+
+import fnmatch
+import os
+import time
+from datetime import datetime, timedelta
+from typing import Any
+
+import numpy as np
+
+from frequenz.sdk.timeseries import Sample
+from frequenz.sdk.timeseries._serializable_ringbuffer import SerializableRingBuffer
+
+FILE_NAME = "ringbuffer.pkl"
+
+FIVE_MINUTES = timedelta(minutes=5)
+
+
+def delete_files_with_prefix(prefix: str) -> None:
+    """Delete all files starting with the given prefix.
+
+    Args:
+        prefix: Prefix of the files to delete
+    """
+    for file in os.listdir():
+        if fnmatch.fnmatch(file, prefix + "*"):
+            os.remove(file)
+
+
+def benchmark_serialization(
+    ringbuffer: SerializableRingBuffer[Any], iterations: int
+) -> float:
+    """Benchmark the given buffer `iteration` times.
+
+    Args:
+        ringbuffer: Ringbuffer to benchmark to serialize.
+        iterations: amount of iterations to run.
+    """
+    total = 0.0
+    for _ in range(iterations):
+        start = time.time()
+        ringbuffer.dump()
+        SerializableRingBuffer.load(FILE_NAME)
+        end = time.time()
+        total += end - start
+        delete_files_with_prefix(FILE_NAME)
+
+    return total / iterations
+
+
+SIZE = 4000_000
+ITERATIONS = 100
```

Comment:
Maybe you want to group all the global constants together on top _and_ have each of these also well documented? `SIZE` and `ITERATIONS` also sounds pretty generic, which makes it hard to figure out the meaning just be looking at them. :)

Reply:
I mean, in the context of a benchmark and a ringbuffer I feel those are _very_ unambigious in what they could mean...